frontend/components/TableContainer/DataTable/TruncatedTextCell/TruncatedTextCell.tsx

```diff
@@ -49,8 +49,9 @@ const TruncatedTextCell = ({
         backgroundColor="#3e4771"
         id={id}
         data-html
+        className={"tooltip"}
       >
-        <span className={`tooltip tooltip-text`}>{value}</span>
```

Comment:
Don't you need these classes for the selectors in the associated stylesheet?

Reply:
I looked through the codebase and those classes aren't being used, I switched to className `truncated-tooltip` and put it on the parent ReactTooltip component when removing this span component to clean up the code.